lib/graphiti/resource/links.rb

```diff
@@ -74,12 +74,12 @@ def allow_request?(request_path, params, action)
         endpoints.any? do |e|
           has_id = params[:id] || params[:data].try(:[], :id)
           path = request_path
-          if [:update, :show, :destroy].include?(context_namespace) && has_id
+          if [:update, :show, :destroy].include?(action) && has_id
```

Comment:
`action` is passed in by the caller; seems appropriate to use it here instead of duplicating the call to `context_namespace`